### PR TITLE
feat: add SLD for GE.Borehole

### PR DESCRIPTION
**Pull request by JohannaOtt**
@Kate-Lyndegaard I wasn't sure if more of the configuration files need to be edited to make the mechnism in place run as expected or if the changes in the config.style file are sufficient. Please let me know if the other config files need to be adapted as well.

Test datasets are available [here](https://haleconnect.com/#/dataset/org/1293/8f13ed41-76db-43ab-89e9-ea85fa78f932/viewServices), [here](https://haleconnect.com/#/dataset/org/1293/fd2a19dc-c721-423c-8b6d-75d28c9c9bcc/viewServices) and [here](https://haleconnect.com/#/dataset/org/1293/baf3bb6d-141c-4016-9d6f-1fb25018488b/viewServices)

SVC-1406

### CODE REVIEW

**Comment by JohannaOtt** [comment]
@Kate-Lyndegaard Thanks for the feedback. From your experience with the generator: Do you think it is faster to edit the respective files manually for this use case or is it faster to set up the Eclipse/xtent option?

**Review by annat2022 — COMMENTED** [review]
@JohannaOtt I tried to review the PR, but not sure if I'm the right one for this as I'm not familiar with all the dependencies needed for the portrayal repository.
I checked the generated se-file and it LGTM
And related to the created commits: Am I missing something or are the changes done in files _config.style_ and _feature-sytles/feature-styles/GE_Borehole.se_ contained in both added commits?

File: config.style [hunk 1]
```diff
@@ -990,6 +990,15 @@ Layer{
...
 }
+Layer{
+	id: "GE_Borehole"
+	registry-id: "http://inspire.ec.europa.eu/layer/GE.Borehole"
+	tags: inspire production
+	Name: "GE.Borehole"
+	Title: en "Boreholes", de "Bohrlöcher"
+	SpatialObjectTypes: http://inspire.ec.europa.eu/schemas/ge-core/4.0:Borehole
+	Styles: GE_Borehole
+}
```
Comment (Kate-Lyndegaard) [inline config.style:1001]: @JohannaOtt In the config.style document you always need to add a layer block (as you have added here), and down at the bottom of this document (line 1137 or after), you also need to add a style block.

**Review by Kate-Lyndegaard — APPROVED** [review]

File: config.style (hunk 1 above)
Comment (Kate-Lyndegaard) [reply config.style:1001]: @JohannaOtt I see you added the Style block in the following commit.

**Comment by Kate-Lyndegaard** [comment]
> @Kate-Lyndegaard Thanks for the feedback. From your experience with the generator: Do you think it is faster to edit the respective files manually for this use case or is it faster to set up the Eclipse/xtent option?

I think it's faster and cleaner if the xtent code is working properly.

**Comment by JohannaOtt** [comment]
Thanks for reviewing everyone!

> Am I missing something or are the changes done in files config.style and feature-sytles/feature-styles/GE_Borehole.se contained in both added commits?

@annat2022 For config.style there is an additional change in the second commit which is a fixup commit. It is a bit confusing now that we keep fixup commits in the PR and only squah them once the PR is reviewed. The review by commit as we usually do it is not as helpful anymore now unfortunately.
For the GE_Borehole.se; There are two files with the same name contained (one in the "generated" folder) and each of it is added in one of the commits. I assume is that is what is causing confusion here.


> I think it's faster and cleaner if the xtent code is working properly.

Ok, then I will try to get that running before adding an SLD file next time. I think for now it is not really worth it to change things now.

**Pull request merged**